client/tests/integration/mod.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(not(coverage))]
```

Comment:
This probably needs to be added to a few more crates. There's a few things that are just not coverable.